src/MBBSEmu.Btrieve/BtrieveFile.cs

```diff
@@ -143,6 +147,16 @@ public ushort KeyCount
             }
         }
 
+        public ushort DupOffset
+        {
+            get => BitConverter.ToUInt16(_fcr, 0x72);
```

Comment:
You can make these expression-bodied members:

`public ushort DupOffset => BitConverter.ToUInt16(_fcr, 0x72);`